MetaData/data/cross_sections.json

```diff
@@ -61,7 +61,8 @@
 	"DYToEE_NNPDF30_13TeV-powheg-pythia8"                           : { "xs" : 2008.33 },
         "DYJetsToLL_M-50_TuneCUETP8M1_13TeV-amcatnloFXFX-pythia8"       : { "xs" : 6025.0, "itype":20                    },
 	"DYJetsToLL_M-50_TuneCUETP8M1_13TeV-madgraphMLM-pythia8"        : { "xs" : 6025.0, "itype":20                    },
-	"DYJetsToLL_M-50_TuneCUETP8M1_13TeV-madgraphMLM-herwigpp_30M"   : { "xs" : 6025.0, "itype":20                    },
+    "DYJetsToLL_M-50_TuneCUETP8M1_13TeV-madgraphMLM-herwigpp_30M"   : { "xs" : 6025.0, "itype":20                    },
+    "DYJetsToLL_M-50_TuneCP5_13TeV-madgraphMLM-pythia8" : {"xs" : 6225.42, "itype":20 },
```

Comment:
Changes in this file should not be part of this PR. Changes here can be reverted

Reply:
This we can discuss

Reply:
Actually, it's fine. The name of the sample was changed upstream, so it's a minor adaptation.